app/models/translation.rb

```diff
@@ -3,6 +3,7 @@
 require 's3_util'
 require 'xml_util'
 
+# rubocop:disable ClassLength
```

Comment:
Yeah... didn't see a better way, instead of making some new modules which I don't know if would be ideal because almost every method is private...